src/tests/commonTests.ts

```diff
@@ -65,8 +65,8 @@ export async function renders(
   const element = await page.find(getTag(componentTagOrHTML));
 
   expect(element).toHaveAttribute(HYDRATED_ATTR);
-  expect(await element.isVisible()).toBe(options?.visible ?? true);
-  expect((await element.getComputedStyle()).display).toBe(options?.display ?? "inline");
+  expect(await element.isVisible()).toBe(options.visible ?? true);
```

Comment:
@julio8a these two lines should be reverted back. Can you revert both of these changes?

Reply:
options are optional so the `?` needs to be there.